service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/cmo/NewCMOUploadedContentProvider.java

```diff
@@ -30,6 +31,7 @@ public CMOReadyToSealTemplate buildTemplate(HearingBooking hearing, Long caseId,
 
     private String subjectLine(HearingBooking hearing, List<Element<Respondent>> respondents,
                                String familyManCaseNumber) {
-        return String.format("%s, %s", buildSubjectLine(familyManCaseNumber, respondents), hearing.toLabel(DATE));
+        return String.format("%s, %s", buildSubjectLine(familyManCaseNumber, respondents),
```

Comment:
Do you think that this should become the default `buildSubjectLineWithHearing`?

Reply:
We can ask business what they think, currently only the CMO callouts include the type of hearing but I think it would be useful to add hearing type to every email callout